apis/management.cattle.io/v3/rke_types.go

```diff
@@ -39,6 +39,8 @@ type RancherKubernetesEngineConfig struct {
 	AddonJobTimeout int `yaml:"addon_job_timeout" json:"addonJobTimeout,omitempty" norman:"default=30"`
 	// Bastion/Jump Host configuration
 	BastionHost BastionHost `yaml:"bastion_host" json:"bastionHost,omitempty"`
+	// Disable Metrics Server addon
+	DisableMetricsServer bool `yaml:"disable_metrics_server" json:"disableMetricsServer,omitempty"`
```

Comment:
why bool? can we make it similar to ingress? so it has a provider and a list of options

Reply:
hmmm, i see, okay will fix